spec/validators/document_category_validator_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+class DummyDocumentCategory
+  include ActiveModel::Model
+  include ActiveModel::Validations
+  validates_with DocumentCategoryValidator
+end
+
+RSpec.describe DocumentCategoryValidator do
+  context 'Attachment' do
+    let(:laa) { create :legal_aid_application }
+    let(:invalid_attachment_type) { 'xxx-zzz' }
+    let(:valid_attachment_types) { DocumentCategoryValidator::VALID_DOCUMENT_TYPES }
+
+    subject { Attachment.create! legal_aid_application: laa, attachment_type: attachment_type }
+
+    context 'valid attachment types' do
+      let(:attachment_type) { valid_attachment_types.sample }
+      it 'does not fail when trying to create a record with an invalid attachment type' do
```

Comment:
Should this be `a valid attachment type` instead of `an invalid attachment type`?

Reply:
Yep, have changed

Reply:
yep